circle.yml

```diff
@@ -33,10 +36,10 @@ deployment:
     commands:
       - NODE_ENV=production npm run deploy
       - cf login -a https://api.cloud.gov -u $CF_GSA_VOTE_USER -p $CF_GSA_VOTE_PASS -o gsa-ffd -s vote-production
-      - cf push
+      - cf zero-downtime-push vote-gov -f manifest.yml
   staging:
     branch: [staging]
     commands:
       - npm run deploy
       - cf login -a https://api.cloud.gov -u $CF_GSA_VOTE_USER -p $CF_GSA_VOTE_PASS -o gsa-ffd -s vote-staging
-      - cf push -f manifest-staging.yml
+      - cf zero-downtime-push vote-gov-staging -f manifest-staging.yml
```

Comment:
It's okay to inherit this way, from what I have [seen here](https://github.com/18F/vote-gov/blob/3ffb3a61a895adb53fb897b288117b50cf25531b/manifest.yml) [and here](https://github.com/18F/vote-gov/blob/3ffb3a61a895adb53fb897b288117b50cf25531b/manifest-staging.yml). The issues I've seen with inheritance is the issue of using multiple `hosts` that are different on one environment from another. Doesn't look like it's a problem here since we'll be using `CNAME`. 👍